Casks/goldendict.rb

```diff
@@ -0,0 +1,21 @@
+cask 'goldendict' do
+  version '1.5.0-RC2'
+  sha256 'bc04acbd5db51d50b8b6262d69117304df96776472b7a1df7ad42dafafe573ff'
+
+  # sourceforge.net/project/goldendict was verified as official when first introduced to the cask
+  url 'https://jaist.dl.sourceforge.net/project/goldendict/early%20access%20builds/MacOS/GoldenDict-1.5.0-RC2-372-gc3ff15f%28Qt_5121%29.dmg'
```

Comment:
```suggestion
  url 'https://downloads.sourceforge.net/project/goldendict/early%20access%20builds/MacOS/GoldenDict-1.5.0-RC2-372-gc3ff15f%28Qt_5121%29.dmg'
```
The actual download server being used depends on your location and curl will follow this redirect.